src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -436,9 +440,23 @@ public DeploymentOptions(DacDeployOptions options)
             // Setting Display names for all dacDeploy options
             SetDisplayNameForOption();
 
+            // Preparing Include Object types options table
+            GetIncludeObjectTypesDictionary();
+
             SetOptions(options);
         }
 
+        /// <summary>
+        /// Gets include objects enum names and values as a dictionary
+        /// Ex: {key: Aggregate, value: 0}
```

Comment:
like I mentioned in the other PR, does ADS need to know about the enum number of each object type? We should only need the name in ADS. If it's needed in in STS for converting before calling the DacServices apis, then a name to enum dictionary can be in STS.

Reply:
Now that the other options have to be converted anyway from the map to DacDeployOptions when coming back from ADS, it would make sense to only send the Schema Object names to ADS, then also do the conversion back to the enum values in STS

Reply:
what about the display names of these objects? Multiple word objects have a different display name from their enum name, like "StoredProcedure" has a space in the display name "Stored Procedure."

Right now, the display names are specified in the schema compare extension. Sql database projects will also need these, so it would be nice if the display names are also coming from STS so they aren't copied in 2 extensions.